tests/e2e/controlplane/control_plane_test.go

```diff
@@ -232,6 +238,37 @@ spec:
 							ShouldNot(ContainSubstring("Reconciliation done"), "Istio Operator is continuously reconciling")
 						Success("Istio Operator stopped reconciling")
 					})
+
+					It("can be deployed bookinfo with sidecar inyection", func(ctx SpecContext) {
```

Comment:
typo: `injection`, but also maybe change to something like `It("injects sidecars into deployed bookinfo")` ?

Reply:
Fixed